k8s.yaml

```diff
@@ -0,0 +1,200 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: git2grid
+  namespace: git2grid
+spec:
+  replicas: 1
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  minReadySeconds: 5
+  template:
+    metadata:
+      labels:
+        app: git2grid
+        role: git2grid
+    spec:
+      containers:
+        - name: git2grid
+          image: billytrend/git2grid:2
+          ports:
+            - containerPort: 3000
+          env:
```

Comment:
Let's break this into a configmap, that way we can have separate maps for prod vs staging